LicensePlateReader/package.json

```diff
@@ -14,12 +14,13 @@
     "in:prod": "npm install -production",
     "in:all": "npm install",
     "zip:package": "node node_modules/camstreamerlib/CreatePackage -i",
-    "zip": "npm-run-all build in:prod zip:package in:all"
+    "zip": "npm-run-all build in:prod zip:package in:all",
+    "build:zip": "npm-run-all build zip:package"
```

Comment:
k cemu tahle zmena? na vyvoj ti vetsinou staci volat jen build a kdyz chces vytvorit zip tak ho chces vytvorit jen s produkcnima balickama..tohle trosku svadi to zavolat a vytvorit zip i s dev zavislostma..

Reply:
Zapomněl jsem, že si ty skripty můžu pouštět i lokálně...